### ISSUE TITLE
Post-Processor not finding files

### ISSUE THREAD

**Issue by idlesHand** [SickChill/SickChill#6215]
Branch/Commit: master  290b22c8c1bb2b16998429022d4e3f7c3b518e66
OS: Linux-4.19.97-v7l+-armv7l-with-debian-10.3
Browser: Chromium
What you did: auto and manual post processing of downloaded files for shows in library
What happened: files are not recognised
What you expected: files to be found and processed

I have checked the qualities being used and even changed some to any to ensure that was not an issue.

Logs: 
```
2020-04-18 18:33:34 DEBUG    POSTPROCESSOR-AUTO :: /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 4 : No processable items found in folder
2020-04-18 18:33:34 DEBUG    POSTPROCESSOR-AUTO :: Processing folder /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 3
2020-04-18 18:33:38 DEBUG    POSTPROCESSOR-AUTO :: /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 3 : No processable items found in folder
2020-04-18 18:33:38 DEBUG    POSTPROCESSOR-AUTO :: Processing folder /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 7
```
```
2020-04-18 23:49:15 INFO     POSTPROCESSOR-MANUAL :: Beginning manual post processing task: /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)
2020-04-18 23:49:15 DEBUG    POSTPROCESSOR-MANUAL :: Processing in folder /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)
2020-04-18 23:49:15 INFO     POSTPROCESSOR-MANUAL :: Processing /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)
2020-04-18 23:49:15 DEBUG    POSTPROCESSOR-MANUAL :: Processing folder /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)
2020-04-18 23:49:58 DEBUG    POSTPROCESSOR-MANUAL :: /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence) : No processable items found in folder
2020-04-18 23:49:58 DEBUG    POSTPROCESSOR-MANUAL :: Processing folder /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 5
2020-04-18 23:50:07 DEBUG    POSTPROCESSOR-MANUAL :: /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 5 : No processable items found in folder
2020-04-18 23:50:07 DEBUG    POSTPROCESSOR-MANUAL :: Processing folder /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 4
2020-04-18 23:50:16 DEBUG    POSTPROCESSOR-MANUAL :: /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 4 : No processable items found in folder
2020-04-18 23:50:16 DEBUG    POSTPROCESSOR-MANUAL :: Processing folder /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 3
2020-04-18 23:50:25 DEBUG    POSTPROCESSOR-MANUAL :: /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 3 : No processable items found in folder
2020-04-18 23:50:25 DEBUG    POSTPROCESSOR-MANUAL :: Processing folder /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 7
2020-04-18 23:50:32 DEBUG    POSTPROCESSOR-MANUAL :: /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 7 : No processable items found in folder
2020-04-18 23:50:32 DEBUG    POSTPROCESSOR-MANUAL :: Processing folder /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 2
2020-04-18 23:50:38 DEBUG    POSTPROCESSOR-MANUAL :: /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 2 : No processable items found in folder
2020-04-18 23:50:38 DEBUG    POSTPROCESSOR-MANUAL :: Processing folder /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 6
2020-04-18 23:50:45 DEBUG    POSTPROCESSOR-MANUAL :: /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 6 : No processable items found in folder
2020-04-18 23:50:45 DEBUG    POSTPROCESSOR-MANUAL :: Processing folder /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 1
2020-04-18 23:50:52 DEBUG    POSTPROCESSOR-MANUAL :: /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 1 : No processable items found in folder
2020-04-18 23:50:52 INFO     POSTPROCESSOR-MANUAL :: Successfully processed
2020-04-18 23:50:52 INFO     POSTPROCESSOR-MANUAL :: Manual post processing task for /media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence) completed

```
while
```
pi@mediapi:~ $ ls /media/general/downloads/30\ Rock\ \(2006\)\ Season\ 1-7\ S01-S07\ \(1080p\ AMZN\ WEBRip\ x265\ HEVC\ 10bit\ AAC\ 5.1\ Silence\)/Season\ [3,4]
'/media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 3':
'30 Rock (2006) - S03E01 - Do-Over (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E01 - Do-Over (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E02 - Believe in the Stars (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E02 - Believe in the Stars (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E03 - The One with the Cast of Night Court (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E03 - The One with the Cast of Night Court (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E04 - Gavin Volure (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E04 - Gavin Volure (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E05 - Reunion (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E05 - Reunion (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E06 - Christmas Special (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E06 - Christmas Special (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E07 - Señor Macho Solo (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E07 - Señor Macho Solo (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E08 - Flu Shot (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E08 - Flu Shot (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E09 - Retreat to Move Forward (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E09 - Retreat to Move Forward (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E10 - Generalissimo (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E10 - Generalissimo (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E11 - St. Valentine'\''s Day (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E11 - St. Valentine'\''s Day (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E12 - Larry King (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E12 - Larry King (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E13 - Goodbye, My Friend (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E13 - Goodbye, My Friend (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E14 - The Funcooker (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E14 - The Funcooker (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E15 - The Bubble (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E15 - The Bubble (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E16 - Apollo, Apollo (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E16 - Apollo, Apollo (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E17 - Cutbacks (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E17 - Cutbacks (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E18 - Jackie Jormp-Jomp (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E18 - Jackie Jormp-Jomp (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E19 - The Ones (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E19 - The Ones (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E20 - The Natural Order (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E20 - The Natural Order (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E21 - Mamma Mia (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E21 - Mamma Mia (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S03E22 - Kidney Now! (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S03E22 - Kidney Now! (1080p AMZN WEBRip x265 Silence).unknown.ignore'

'/media/general/downloads/30 Rock (2006) Season 1-7 S01-S07 (1080p AMZN WEBRip x265 HEVC 10bit AAC 5.1 Silence)/Season 4':
'30 Rock (2006) - S04E01 - Season 4 (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E01 - Season 4 (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E02 - Into the Crevasse (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E02 - Into the Crevasse (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E03 - Stone Mountain (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E03 - Stone Mountain (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E04 - Audition Day (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E04 - Audition Day (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E05 - The Problem Solvers (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E05 - The Problem Solvers (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E06 - Sun Tea (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E06 - Sun Tea (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E07 - Dealbreakers Talk Show #0001 (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E07 - Dealbreakers Talk Show #0001 (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E08 - Secret Santa (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E08 - Secret Santa (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E09 - Klaus and Greta (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E09 - Klaus and Greta (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E10 - Black Light Attack! (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E10 - Black Light Attack! (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E11 - Winter Madness (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E11 - Winter Madness (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E12 - Verna (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E12 - Verna (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E13 - Anna Howard Shaw Day (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E13 - Anna Howard Shaw Day (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E14 - Future Husband (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E14 - Future Husband (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E15 - Don Geiss, America, and Hope (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E15 - Don Geiss, America, and Hope (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E16 - Floyd (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E16 - Floyd (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E17 - Lee Marvin vs. Derek Jeter (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E17 - Lee Marvin vs. Derek Jeter (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E18 - Khonani (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E18 - Khonani (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E19 - Argus (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E19 - Argus (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E20 - The Moms (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E20 - The Moms (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E21 - Emmanuelle Goes to Dinosaur Land (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E21 - Emmanuelle Goes to Dinosaur Land (1080p AMZN WEBRip x265 Silence).unknown.ignore'
'30 Rock (2006) - S04E22 - I Do Do (1080p AMZN WEBRip x265 Silence).mkv'
'30 Rock (2006) - S04E22 - I Do Do (1080p AMZN WEBRip x265 Silence).unknown.ignore'
```

also from another torrent
Auto
```
2020-04-18 17:25:14 DEBUG    POSTPROCESSOR-AUTO :: Processing folder /media/general/downloads/American Dad! (2005) Season 15 S15 (1080p AMZN WEB-DL x265 HEVC 10bit EAC3 5.1 ImE)
2020-04-18 17:25:18 DEBUG    POSTPROCESSOR-AUTO :: /media/general/downloads/American Dad! (2005) Season 15 S15 (1080p AMZN WEB-DL x265 HEVC 10bit EAC3 5.1 ImE) : No processable items found in folder
```
actual dir contents
```
pi@mediapi:~ $ ls /media/general/downloads/American\ Dad\!\ \(2005\)\ Season\ 15\ S15\ \(1080p\ AMZN\ WEB-DL\ x265\ HEVC\ 10bit\ EAC3\ 5.1\ ImE\)/
'American Dad! (2005) - S15E01 - Santa, Schmanta (1080p AMZN WEB-DL x265 ImE).failed_rename.ignore'
'American Dad! (2005) - S15E01 - Santa, Schmanta (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E02 - Paranoid_Frandroid (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E03 - The Census of the Lambs (1080p AMZN WEB-DL x265 ImE).failed_rename.ignore'
'American Dad! (2005) - S15E03 - The Census of the Lambs (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E04 - Shell Game (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E05 - The Mural of the Story (1080p AMZN WEB-DL x265 ImE).failed_rename.ignore'
'American Dad! (2005) - S15E05 - The Mural of the Story (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E06 - (You Gotta) Strike for Your Right (1080p AMZN WEB-DL x265 ImE).failed_rename.ignore'
'American Dad! (2005) - S15E06 - (You Gotta) Strike for Your Right (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E07 - Klaustastrophe.tv (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E08 - Death by Dinner Party (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E09 - The Never-Ending Stories (1080p AMZN WEB-DL x265 ImE).failed_rename.ignore'
'American Dad! (2005) - S15E09 - The Never-Ending Stories (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E10 - Railroaded (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E11 - My Purity Ball and Chain (1080p AMZN WEB-DL x265 ImE).failed_rename.ignore'
'American Dad! (2005) - S15E11 - My Purity Ball and Chain (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E12 - OreTron Trail (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E13 - Mean Francine (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E14 - One-Woman Swole (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E15 - Flavortown (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E16 - Persona Assistant (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E17 - The Legend of Old Ulysses (1080p AMZN WEB-DL x265 ImE).failed_rename.ignore'
'American Dad! (2005) - S15E17 - The Legend of Old Ulysses (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E18 - Twinanigans (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E19 - Top of the Steve (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E20 - Funnyish Games (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E21 - Fleabiscuit (1080p AMZN WEB-DL x265 ImE).mkv'
'American Dad! (2005) - S15E22 - The Future is Borax (1080p AMZN WEB-DL x265 ImE).mkv'
```
final example

```
2020-04-18 16:55:24 DEBUG    POSTPROCESSOR-AUTO :: Processing folder /media/general/downloads/Atlanta (2016) Season 1 S01 (1080p AMZN WEB-DL x265 HEVC 10bit EAC3 5.1 RZeroX)
2020-04-18 16:55:26 DEBUG    POSTPROCESSOR-AUTO :: /media/general/downloads/Atlanta (2016) Season 1 S01 (1080p AMZN WEB-DL x265 HEVC 10bit EAC3 5.1 RZeroX) : No processable items found in folder
```
```
2020-04-18 23:44:37 INFO     POSTPROCESSOR-MANUAL :: Beginning manual post processing task: /media/general/downloads/Atlanta (2016) Season 2 S02 (1080p AMZN WEB-DL x265 HEVC 10bit EAC3 6.0 RZeroX)
2020-04-18 23:44:37 DEBUG    POSTPROCESSOR-MANUAL :: Processing in folder /media/general/downloads/Atlanta (2016) Season 2 S02 (1080p AMZN WEB-DL x265 HEVC 10bit EAC3 6.0 RZeroX)
2020-04-18 23:44:37 INFO     POSTPROCESSOR-MANUAL :: Processing /media/general/downloads/Atlanta (2016) Season 2 S02 (1080p AMZN WEB-DL x265 HEVC 10bit EAC3 6.0 RZeroX)
2020-04-18 23:44:37 DEBUG    POSTPROCESSOR-MANUAL :: Processing folder /media/general/downloads/Atlanta (2016) Season 2 S02 (1080p AMZN WEB-DL x265 HEVC 10bit EAC3 6.0 RZeroX)
```

file contents
```
pi@mediapi:~ $ ls /media/general/downloads/Atlanta\ \(2016\)\ Season\ 2\ S02\ \(1080p\ AMZN\ WEB-DL\ x265\ HEVC\ 10bit\ EAC3\ 6.0\ RZeroX\)/
'Atlanta (2016) - S02E01 - Alligator Man (1080p AMZN WEB-DL x265 RZeroX).mkv'
'Atlanta (2016) - S02E01 - Alligator Man (1080p AMZN WEB-DL x265 RZeroX).unknown.ignore'
'Atlanta (2016) - S02E02 - Sportin'\'' Waves (1080p AMZN WEB-DL x265 RZeroX).mkv'
'Atlanta (2016) - S02E03 - Money Bag Shawty (1080p AMZN WEB-DL x265 RZeroX).mkv'
'Atlanta (2016) - S02E04 - Helen (1080p AMZN WEB-DL x265 RZeroX).mkv'
'Atlanta (2016) - S02E05 - Barbershop (1080p AMZN WEB-DL x265 RZeroX).mkv'
'Atlanta (2016) - S02E06 - Teddy Perkins (1080p AMZN WEB-DL x265 RZeroX).mkv'
'Atlanta (2016) - S02E07 - Champagne Papi (1080p AMZN WEB-DL x265 RZeroX).mkv'
'Atlanta (2016) - S02E08 - Woods (1080p AMZN WEB-DL x265 RZeroX).mkv'
'Atlanta (2016) - S02E09 - North of the Border (1080p AMZN WEB-DL x265 RZeroX).mkv'
'Atlanta (2016) - S02E10 - FUBU (1080p AMZN WEB-DL x265 RZeroX).mkv'
'Atlanta (2016) - S02E11 - Crabs in a Barrel (1080p AMZN WEB-DL x265 RZeroX).mkv'

```

**Comment by miigotu** [comment]
Whatever you are using to rename them, is adding .ignore to file names which prevents the dir from being processed.

**Comment by miigotu** [comment]
That is couchpotato doing that, you need to use separate download folders for SC and Couchpotato

**Comment by miigotu** [comment]
Also, do you have hevc support enabled?

**Comment by idlesHand** [comment]
I didn't think the .ignore files affected sickchill so I used find to delete all the ignore files within the 30 rock example and still got the same result in debug log. Plus I checked a download that was processed successfully and it too contained the .ignore though I can't say which came first, processing or couchpotato adding .ignore.

Also I have been sure to check the x265 / hevc is enabled in the search settings.
